cxx-squid/src/main/java/org/sonar/cxx/preprocessor/CxxPreprocessor.java

```diff
@@ -743,19 +743,20 @@ private int matchArguments(List<Token> tokens, List<Token> arguments) {
           break;
         }
       } while (true);
-
+    } catch (MismatchException me) {
+      LOG.debug("Mismatch: expected ','got: '" + rest.get(0).getValue() + "'");
```

Comment:
ok, but then do we even need to do the debug print. if running with -X it will just print loads of not needed stuff. right?